src/status_im/common/privacy/view.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.contexts.onboarding.privacy.view
+(ns status-im.common.privacy.view
```

Comment:
Moved from `oboarding` to `common` since we reuse it, this change has also been applied to the terms of use too.